### ISSUE TITLE
[Sonic Adventure DX 2004] Error when trying to use reshade preventing boot.

### ISSUE THREAD

**Issue by PrecursorsLegacyDev** [crosire/d3d8to9#117]
![Capture](https://user-images.githubusercontent.com/41239454/93754325-c8d54700-fbbe-11ea-8781-4a447e438796.PNG)
I recieve this error when running sonic adventure dx with reshade. Runs perfectly fine without the dll if not using rehsade. Unsure of how to fix it.

**Comment by michael-fadely** [comment]
Are you using my Lantern Engine mod ([sadx-dc-lighting](https://github.com/michael-fadely/sadx-dc-lighting))? It does some hooking which might cause problems. Or any mods, for that matter?

**Comment by PrecursorsLegacyDev** [comment]
yes i'm using the sadx lantern mod. all in all it's the sadx dreamcast mod

**Comment by crosire** [comment]
Closing since unrelated to d3d8to9.
